app/src/main/java/com/example/happybirthday/MainActivity.kt

```diff
@@ -37,25 +48,65 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    Greeting("Android")
+                    BirthdayGreetingWithImage(
+                        stringResource(R.string.happy_birthday_text),
+                        stringResource(R.string.signature_text)
+                    )
                 }
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
+fun BirthdayGreetingWithText(message: String, from: String, modifier: Modifier = Modifier) {
+    // Create a column so that texts don't overlap
+    Column(
         modifier = modifier
-    )
+            .fillMaxSize()
+            .padding(8.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = message,
+            fontSize = 100.sp,
+            lineHeight = 110.sp,
```

Comment:
per message in the chat, I think we can remove lineHeight because it seems to be an issue only with the preview

Reply:
Discussed and I can reproduce. We may need to specify the line height now in M3 so I will resolve for now.